.cargo/config_fast_build

```diff
@@ -0,0 +1,23 @@
+# Rename this file to `config.toml` to enable "fast build" configuration. Please read the notes below.
```

Comment:
@wmorgue where did this file come from?

Reply:
Any news?
cc @dashed

Reply:
@wmorgue this is an app used internally by a team, any changes need to be reviewed by the team. Currently, the demo meets our needs.

We recommend you follow https://docs.sentry.io/platforms/rust/ for instructions on how to use the Sentry Rust SDK, as your app and environment may differ from what we're doing in sentry-demos. Appreciate your idea for reducing the dependencies.

What is this change for?
```
# NOTE: For maximum performance, build using a nightly compiler
# If you are using rust stable, remove the "-Zshare-generics=y" below.

[target.x86_64-unknown-linux-gnu]
linker = "/usr/bin/clang"
rustflags = ["-Clink-arg=-fuse-ld=lld", "-Zshare-generics=y"]

# NOTE: you must manually install https://github.com/michaeleisel/zld on mac. you can easily do this with the "brew" package manager:
# `brew install michaeleisel/zld/zld`
[target.x86_64-apple-darwin]
rustflags = ["-C", "link-arg=-fuse-ld=/usr/local/bin/zld", "-Zshare-generics=y", "-Zrun-dsymutil=no"]

[target.x86_64-pc-windows-msvc]
linker = "rust-lld.exe"
rustflags = ["-Zshare-generics=y"]

# Optional: Uncommenting the following improves compile times, but reduces the amount of debug info to 'line number tables only'
# In most cases the gains are negligible, but if you are on macos and have slow compile times you should see significant gains.
#[profile.dev]
#debug = 1
```

Reply:
This change is optional and is intended to build the project faster when there are a large number of dependencies.